CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
```

Comment:
I think the Releases section on GitHub is better than a changelog file in this case. If you want to keep track of the changes for a new version you can make a draft.